symfony/messenger/4.3/config/packages/messenger.yaml

```diff
@@ -6,7 +6,7 @@ framework:
         transports:
             # https://symfony.com/doc/current/messenger.html#transport-configuration
             # async: '%env(MESSENGER_TRANSPORT_DSN)%'
-            # failed: 'doctrine://default?queue_name=failed'
+            # failed: '%env(MESSENGER_TRANSPORT_DSN)%?queue_name=failed'
```

Comment:
Using the same env variable for both the "working queue" and the "failed queue" is confusing.
I' rather use something like "MESSENGER_TRANSPORT_FAILED_DSN"

Reply:
In my opinion, using the same environment variable for both queues is actually a good default, because most of the times, only the queue name is different. I think that it makes sense to configure the queue name in the configuration file, and the transport 'technology' with an environment variable.

Reply:
nvm, I missed the `?queue_name=failed` in the parameter